android/jni/include.mk

```diff
@@ -189,7 +192,7 @@ $(foreach schema,$(1),\
   $(call flatbuffers_header_build_rule,\
     $(schema),$(strip $(2)),$(strip $(3)),$(strip $(4))))\
 $(foreach src,$(strip $(5)),\
-  $(eval $(PORTABLE_LOCAL_PATH)$$(src): \
+  $(eval $(call local-source-file-path,$(src)): \
```

Comment:
I added PORTABLE_LOCAL_PATH at some point because Windows ndk-build only works with relative paths. Does this change work on Windows?